spec/src/main/asciidoc/repository.asciidoc

```diff
@@ -539,8 +539,9 @@ Or you can obtain the next (or previous) page relative to a known entity,
 [source,java]
 ----
 Customer c = ...
-PageRequest p = PageRequest.ofSize(50)
-                           .afterKey(c.lastName, c.firstName, c.id);
+PageRequest p = PageRequest.ofPage(10)
```

Comment:
```suggestion
PageRequest p = PageRequest.ofSize(50)
````

Reply:
Your suggestion would result in duplicating the size,
```
PageRequest p = PageRequest.ofSize(50).size(50).afterCursor...
```
so we shouldn't add this change exactly as proposed but could still remove the page if we decide to.

Reply:
Nope, remove the size.

My question is: in this demo with the cursor, do we need the page?

Reply:
> do we need the page?

Yes you need it because backward navigation from the (approximate) first page works differently from backward navigation from any other page. So setting `page` to `1` when you are not at the first page results in weird behavior.